### PR TITLE
Flux 0.138.0 and 0.139.0

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/flux/v0.x/release-notes.md [hunk 1]
```diff
@@ -10,6 +10,30 @@ aliases:
   - /influxdb/cloud/reference/release-notes/flux/
 ---
 
+## v0.139.0 [2021-11-01]
+
+### Features
+- Always continue type inference on error.
```
Comment (kelseiv) [inline content/flux/v0.x/release-notes.md:16]: may read a bit like a bug fix... could we say Add support for type inference on error? would this apply to all functions?

File: content/flux/v0.x/release-notes.md [hunk 2]
```diff
@@ -10,6 +10,30 @@ aliases:
...
+
+### Bugfix
```
Comment (kelseiv) [inline content/flux/v0.x/release-notes.md:18]:
```suggestion
### Bug fixes
```

File: content/flux/v0.x/stdlib/experimental/bitwise/_index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+The Flux experimental `bitwise` package provides functions for performing bitwise
+operations on integers and unsigned integers.
+Import the `experimental/bitwise` package:
+
+```js
+import "experimental/bitwise"
+```
+
+## Functions
+Functions prefixed with `s` operate on [signed integers (int)](/flux/v0.x/data-types/basic/int).
```
Comment (kelseiv) [inline content/flux/v0.x/stdlib/experimental/bitwise/_index.md:27]: would it make sense to link to the operate part?: /flux/v0.x/data-types/basic/int/#operate-on-integers

**Review by kelseiv — APPROVED** [review]

File: content/flux/v0.x/stdlib/experimental/bitwise/_index.md (hunk 3 above)
Comment (sanderson) [reply content/flux/v0.x/stdlib/experimental/bitwise/_index.md:27]: Good callout. I should add examples for performing bitwise operations with integers.

File: content/flux/v0.x/release-notes.md (hunk 1 above)
Comment (sanderson) [reply content/flux/v0.x/release-notes.md:16]: Yes, this applies to all functions. I'll reword this release not a bit. The reason this is a feature is Flux will now report all type inference errors in a script. Before, it'd stop on the first error and report it. You'd then have to run the script again to get to the next error; then again for the next error. This just reports everything in one shot. Super nice.

**Pull request merged**